modules/ROOT/pages/publish-api-project-to-exchange.adoc

```diff
@@ -0,0 +1,19 @@
+= Publish an API Project to Exchange
+
+You can publish your API project as a new version of an API specification to Exchange.
+
+To publish your API project as an API in Exchange:
+
+. In the Package Explorer view, right-click your API specification project and select *Manage API Specification* > *Publish to Exchange*.
++
+image::publish-api-project-to-exchange.png[]
+. Configure the following properties for your API to be an Exchange asset:
++
+image::configure-api-spec.png[]
+<1> In *Specification name*, type the name of your API
```

Comment:
I left [a longish comment in a PR](https://github.com/mulesoft/docs-anypoint-datagraph/pull/7) about this yesterday...I read the image guidelines not to use a numbered list to describe annotations